doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+
+  # Mount the Clear Linux OS root filesystem on the SATA disk
+  $ sudo mount /dev/sda3 /mnt
+  $ sudo cp -r <kernel-modules-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+  $ sudo umount /mnt
+
+  # Mount the Clear Linux OS root filesystem on the USB flash disk
+  $ sudo mount /dev/sdb3 /mnt
+  $ sudo cp -r <path-to-kernel-module-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+  $ sudo umount /mnt
+
+Step 4: Copy the bootable kernel image to the /boot directory::
+
+  $ sudo cp <path-to-kernel-image-built-in-step1>/bzImage /boot/
+
+
+Note: If you update kernel image (bzImage) and modules on ubuntu of KBL NUC i7, need use scp to copy it from the remote host building machine.
+If you install EBtool Docker on the ubuntu of KBL NUC i7, it will be easy to update kernel image and modules.
+
+Update ACRN hypervisor Image
+****************************
+
+Step 1: Before building the ACRN hypervisor, you need to figure out the I/O address of the serial port, and the PCI BDF addresses of the SATA controller and the USB controllers on KBL NUC i7 board.
+Enter the following command to get the I/O addresses of the serial ports. KBL NUC i7 boards support one serial port ttyS0. Need to connect the serial port to the development host, in order to access the ACRN serial console to switch between pre-launched VMs::
+
+    $dmesg | grep ttyS0
+    [    0.000000] console [ttyS0] enabled
+    [    1.562546] 00:01: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
+    The following command prints detailed information about all PCI buses and devices in the system.
+    $lspci -vv
+    00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
+            Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
+    00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
+            Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+    00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+            Subsystem: Intel Corporation Ethernet Connection I219-LM
+    Note: Double check PCI devices BDF defined in the "hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h".
+
+Step 2:Clone the ACRN source code and configure the build options with make menuconfig command::
+
+  $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+  $ cd acrn-hypervisor/hypervisor
+  $ make menuconfig
```

Comment:
Please use "make menuconfig BOARD=nuc7i7bnh" instead of "make menuconfig", and double check the following configuration items. Thanks!

Reply:
If I followed you correctly, you are building the hypervisor in Ubuntu. If true, we should also refer to the "Build ACRN from source" (using :ref:`getting-started-building`) for further instructions on how to correctly set-up Ubuntu to build ACRN (as there are a few tricks needed if using 18.04 or 16.04)

Reply:
@gvancuts Thanks, let's follow "Build ACRN from source" and refer it. use "make defconfig BOARD=nuc7i7bnh" and "make menuconfig" here.

Reply:
@gvancuts thanks! let's follow "Build ACRN from source" and refer it; use "make defconfig BOARD=nuc7i7bnh" and "make menuconfig"

Reply:
fix at doc v2 version

Reply:
fix at doc v2 version